### PR TITLE
added auth routing tree with some dummy templates data

### PR DESCRIPTION
**Pull request by HellraiserCW**
- auth routing tree;
- dummy data in templates;
- auth service with no logic.

### CODE REVIEW

File: src/app/app.config.ts [hunk 1]
```diff
@@ -1,4 +1,9 @@
-export const enum APP_ROUTER_NAME {
+export enum APP_ROUTER_NAME {
   auth = 'auth',
-  main = 'main'
+  main = 'main',
+  forgotPass = 'forgot-password',
+  credentials = 'credentials',
+  about = 'about',
+  interests = 'interests',
+  socials = 'socials'
```
Comment (Ponomaleks) [inline src/app/app.config.ts:8]: let's write enum keys in title case as in TS documentation

File: src/app/auth/container/auth.component.html [hunk 2]
```diff
@@ -1 +1,14 @@
...
+<p>Sign In</p>
+<div>
+  <input type="email" />
+  <br />
+  <input type="password" />
+  <br />
+  <button (click)="onSignIn()">Sign In</button>
+  <br />
+  <a [routerLink]="['/', APP_ROUTER_NAME.auth, APP_ROUTER_NAME.forgotPass]">Forgot password?</a>
+  <hr />
```
Comment (GennadiyShpak) [inline src/app/auth/container/auth.component.html:11]: Please use scss if you want add some styles

File: src/app/auth/container/auth.component.html [hunk 3]
```diff
@@ -1 +1,14 @@
 <p>auth works!</p>
+<p>Sign In</p>
+<div>
```
Comment (GennadiyShpak) [inline src/app/auth/container/auth.component.html:3]: Looks like it must be form tag

File: src/app/auth/container/auth.component.ts [hunk 4]
```diff
@@ -1,11 +1,40 @@
...
   templateUrl: './auth.component.html',
   styleUrls: ['./auth.component.scss']
 })
-export class AuthComponent {}
+export class AuthComponent {
+  readonly APP_ROUTER_NAME = APP_ROUTER_NAME;
+
+  private authService = inject(AuthService);
+  private router = inject(Router);
+  private route = inject(ActivatedRoute);
```
Comment (GennadiyShpak) [inline src/app/auth/container/auth.component.ts:21]: why do we need inject function?

File: src/app/auth/container/auth.component.ts [hunk 5]
```diff
@@ -1,11 +1,40 @@
...
+      .subscribe();
+  }
+
+  onSignUp(): void {
+    this.router.navigate([APP_ROUTER_NAME.credentials], { relativeTo: this.route });
+  }
+
+  onForgotPassword(): void {
+    this.router.navigate([APP_ROUTER_NAME.forgotPass], { relativeTo: this.route });
+  }
```
Comment (GennadiyShpak) [inline src/app/auth/container/auth.component.ts:39]: why do we need this logic?

File: src/app/auth/index.ts [hunk 6]
```diff
@@ -0,0 +1 @@
+export * from './services/auth.service';
```
Comment (GennadiyShpak) [inline src/app/auth/index.ts:1]: Do we want to use this pattern?

File: src/app/auth/pages/about/about.component.html [hunk 7]
```diff
@@ -0,0 +1,22 @@
+<p>about works!</p>
+<p>Step 2 . Tell the world about you</p>
+<div>
+  <label for="about">Tell us a few words about yourself </label>
+  <br />
+  <textarea id="about"></textarea>
+</div>
+<div>
+  <label for="city">City:</label>
+  <br />
```
Comment (GennadiyShpak) [inline src/app/auth/pages/about/about.component.html:10]: bad idea

File: src/app/auth/pages/credentials/credentials.component.ts [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+@Component({
+  selector: 'epm-credentials',
+  standalone: true,
+  imports: [CommonModule, RouterLink],
+  templateUrl: './credentials.component.html',
+  styleUrls: ['./credentials.component.scss']
+})
+export class CredentialsComponent {
+  readonly APP_ROUTER_NAME = APP_ROUTER_NAME;
```
Comment (GennadiyShpak) [inline src/app/auth/pages/credentials/credentials.component.ts:15]: don`t worget write type of property

File: src/app/auth/pages/forgot-password/forgot-password.component.html [hunk 9]
```diff
@@ -0,0 +1,6 @@
+<p>forgot-password works!</p>
+<p>Enter your email to reset password</p>
+<input type="email" />
+<br />
```
Comment (GennadiyShpak) [inline src/app/auth/pages/forgot-password/forgot-password.component.html:4]: my eyes

File: src/app/auth/services/auth.service.ts [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+export class AuthService {
+  isSignedIn = false;
+
+  signIn(): Observable<boolean> {
+    return of(true).pipe(
+      tap(value => {
+        this.isSignedIn = value;
+      })
+    );
+  }
```
Comment (GennadiyShpak) [inline src/app/auth/services/auth.service.ts:16]: why do we need this?

File: src/app/app.config.ts (hunk 1 above)
Comment (HellraiserCW) [reply src/app/app.config.ts]: Rewrote with titlecase.

File: src/app/auth/container/auth.component.html (hunk 2 above)
Comment (HellraiserCW) [reply src/app/auth/container/auth.component.html]: dummy data deleted, yagni.

File: src/app/auth/container/auth.component.html (hunk 3 above)
Comment (HellraiserCW) [reply src/app/auth/container/auth.component.html]: +

File: src/app/auth/container/auth.component.ts (hunk 5 above)
Comment (HellraiserCW) [reply src/app/auth/container/auth.component.ts]: dummy data deleted, yagni.

File: src/app/auth/pages/about/about.component.html (hunk 7 above)
Comment (HellraiserCW) [reply src/app/auth/pages/about/about.component.html]: dummy data deleted, yagni.

File: src/app/auth/pages/credentials/credentials.component.ts (hunk 8 above)
Comment (HellraiserCW) [reply src/app/auth/pages/credentials/credentials.component.ts]: added property definition

File: src/app/auth/pages/forgot-password/forgot-password.component.html (hunk 9 above)
Comment (HellraiserCW) [reply src/app/auth/pages/forgot-password/forgot-password.component.html]: dummy data deleted, yagni.

File: src/app/auth/services/auth.service.ts (hunk 10 above)
Comment (HellraiserCW) [reply src/app/auth/services/auth.service.ts]: dummy data deleted, yagni.

**Pull request closed without merging**